src/components/ContentBoxes.tsx

```diff
@@ -54,18 +54,24 @@ const ContentBoxes = ({
       justifyContent: 'center'
     }}>
     <div style={{ maxWidth: 1140 }}>
-      <Heading.h1 style={{ textAlign: 'center' }}>{mainTitle}</Heading.h1>
+      <Heading.h1
+        style={{
+          textAlign: 'left',
+          margin: `0 0 0 ${suomifiTheme.spacing.m}`
+        }}>
+        {mainTitle}
+      </Heading.h1>
       <div
         style={{
           display: 'flex',
           flexWrap: 'wrap',
           justifyContent: 'center',
-          margin: `${suomifiTheme.spacing.l} 0`
+          margin: `${suomifiTheme.spacing.l} 0 0 0`
         }}>
         <div
           style={{
             width: '100%',
-            maxWidth: wrapAll ? 800 : 1140,
+            maxWidth: wrapAll ? 900 : 1140,
```

Comment:
Would be nice to have these also stored somewhere then used here and the also responsive mobile breakpoints used with react-reponsive. Or are they based on how things look and calculated from combination of elements in the view?

Reply:
I think this has just been a lazy way of getting the page to look like it should. The new values are doing the same... I'll look into finding a better place for these values at some point. Requires some studying on my part though.

Reply:
Yeah this should be left for later on if it is done like custom CSS way here.